library/common-library/DEVELOPERS.md

```diff
@@ -189,9 +189,10 @@ we are going to use during the documentation of these functions:
 
 ```yaml
 global:
-  registry: nexus-3-instance.internal.clients-domain.tld
+  images:
+    registry: nexus-3-instance.internal.clients-domain.tld
 jobImage:
-  registry: k8s.gcr.io
+  registry: # defaults to "example.tld" when empty in these examples
```

Comment:
I have a question. Here you mean it defaults to `nexus-...`, isn't it?

Reply:
No, this is the in the docs, not the tests: https://github.com/newrelic/helm-charts/pull/931/files/4fcdd11cada8d32ff938e3a9768662930414d16b#diff-ead97ed97e34739841bc3318ba1886dbdeb49568c970a273c57acc8f7001c2a4R227